Tools/KeyCloak/Terraform/variables.tf

```diff
@@ -130,6 +130,26 @@ variable "client_hg_mobile" {
   description = "HealthGateway mobile client configuration"
 }
 
+variable "client_hg_mobile_android" {
```

Comment:
Since these two variable blocks are identical to the mobile block we already have and should never have actual differences I recommend you remove these and use the existing variables.